patches/openwrt/0006-lantiq-xrx200-make-DSA-images-swconfig-upgradable.patch

```diff
@@ -0,0 +1,30 @@
+From: David Bauer <mail@david-bauer.net>
+Date: Tue, 5 Jul 2022 23:49:31 +0200
+Subject: lantiq-xrx200: make DSA images swconfig upgradable
+
+diff --git a/target/linux/lantiq/image/tp-link.mk b/target/linux/lantiq/image/tp-link.mk
+index 756105b94c7f844439132ab57837c9ac87d231ca..5563968de9ab3f491fa89d755fcc6e6fff79948f 100644
+--- a/target/linux/lantiq/image/tp-link.mk
++++ b/target/linux/lantiq/image/tp-link.mk
+@@ -1,7 +1,7 @@
+ DEVICE_VARS += TPLINK_FLASHLAYOUT TPLINK_HWID TPLINK_HWREV TPLINK_HWREVADD TPLINK_HVERSION
+ 
+ define Device/dsa-migration
+-  DEVICE_COMPAT_VERSION := 1.1
++  DEVICE_COMPAT_VERSION := 1.0
+   DEVICE_COMPAT_MESSAGE := Config cannot be migrated from swconfig to DSA
```

Comment:
aren't these "messages" obsolete then for both mk files?

Reply:
They are not included to the metadata in case the Compat-version is set to 1.0.